frontend/src/components/NewItemCard.tsx

```diff
@@ -147,13 +178,30 @@ export default function NewItemCard(props: Props) {
                     />
                 </label>
                 <label>
-                    <p>Image:</p>
-                    <input
-                        type="text"
-                        value={itemImg}
-                        placeholder="image"
-                        onChange={(e) => setItemImg(e.target.value)}
-                    />
+                    <div className="image-dialogue-container">
+
+{/*
+                        {image && <img src={URL.createObjectURL(image)} alt={"image not found"}/>}
+*/}
+
+
+                        {itemImg && <img src={itemImg} alt={"image not found"}/>}
```

Comment:
`{itemImg && <img src={itemImg} alt={"image not found"}`
Ich bin mir nicht sicher, ob die Logik so einwandfrei ist. Wenn itemImg leer (null) ist, dann wird doch der hintere Teil (hinter &&) nicht mehr ausgeführt, oder? aber dann steht da bei alt "image not found". Müsste hier dann nicht z.B. stehen "Preview of the selected image" (oder ähnliches)?

Reply:
Ja genau, nach dem ?? wird nur ausgeführt wenn was im itemImg drinsteht, nur der link muss nicht unbedingt aktuell sein glaube ich. daher das alt.